experimental/devices/ccs811/doc.go

```diff
@@ -0,0 +1,12 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+// Package CCS811 controls CCS811 Volatile Organic Compounds sensor via
+// I²C interface.
+//
+// Datasheet
+//
+// https://ams.com/ccs811
```

Comment:
That's the product page, the datasheet is at https://ams.com/documents/20143/36005/CCS811_DS000459_6-00.pdf. You can put both.